pandas/_testing/__init__.py

```diff
@@ -485,12 +485,12 @@ def getPeriodData(nper=None):
 # make frame
 def makeTimeDataFrame(nper=None, freq="B"):
     data = getTimeSeriesData(nper, freq)
-    return DataFrame(data)
+    return DataFrame(data)._consolidate()
```

Comment:
This change should in theory no longer be needed? (assuming this was done to ensure consolidated dataframe when the default was changed to not copy, and thus result in non-consolidated dataframe)

(and same for the 2 cases just below)